Packs/FortiSIEM/IncidentFields/incidentfield-Incident_First_Seen.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Please add a prefix to the field name, cliname and id (like the other fields). You can consider using the common field `firstseen` instead as well (please note that this field is `shortText` and not `date`)

Reply:
Done